### PR TITLE
Add new theme Light and Dark

### PR DESCRIPTION
**Pull request by inonehp**

### CODE REVIEW

File: colorschemes/dark-abc.conf [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#
```
Comment (eht16) [inline colorschemes/dark-abc.conf:1]: Would you mind adding a full file header with license and copyright information?
Like this one: https://github.com/geany/geany-themes/blob/master/colorschemes/github.conf

File: colorschemes/dark-abc.conf [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+# GNU General Public License
+#
+#https://github.com/geany/geany/blob/d9f8cdbad58d09f0c18ca8acccb49209263018f0/data/filedefs/filetypes.common#L2
+# Inspired by Github, Sublime Text, Geany
+
+
+
+[theme_info]
+name=Dark Abc
```
Comment (eht16) [inline colorschemes/dark-abc.conf:10]: The other themes with light/dark variants mostly use a naming scheme like "<name>-light" and "<name>-dark".

What do you think about following this scheme and rename your themes to "abc-light" and "abc-dark"?

(then also please rename the files and screenshots)

**Review by eht16 — CHANGES_REQUESTED** [review]
Could you please add your themes to the `AUTHORS` file?

File: colorschemes/dark-abc.conf (hunk 2 above)
Comment (inonehp) [reply colorschemes/dark-abc.conf:10]: Sorry, I didn't rename it, I uploaded it with the new name and deleted it with the old name

**Comment by eht16** [comment]
@inonehp 
Thanks for the changes, to me it looks good now.
If you are fine it with, I would like to merge this so it is ready to use for the upcoming Geany 2.0 release.

**Review by eht16 — APPROVED** [review]

**Pull request merged**